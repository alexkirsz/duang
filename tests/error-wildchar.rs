use duang::duang;

duang!(
pub fn foo((_a,_):(i32,i32)) {}
);

fn main() {
  foo!((1,2));
}
