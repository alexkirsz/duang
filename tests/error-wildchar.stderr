error: expected expression, found reserved identifier `_`
 --> $DIR/error-wildchar.rs:4:16
  |
4 | pub fn foo((_a,_):(i32,i32)) {}
  |                ^ expected expression
...
8 |   foo!((1,2));
  |   ------------ in this macro invocation
